modules.d/80test-root/test-init.sh

```diff
@@ -39,7 +39,7 @@ echo "made it to the rootfs!"
 if getargbool 0 rd.shell; then
     strstr "$(setsid --help)" "control" && CTTY="-c"
     # shellcheck disable=SC2086
```

Comment:
`shellcheck disable=SC2086` can be removed as part of this PR.

Reply:
removed.